menas/src/test/scala/za/co/absa/enceladus/menas/package.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus
+
+package object menas {
+
+  /**
+   * Resource paths of files for schema testing
+   */
+  object TestResourcePath {
```

Comment:
Putting constants in nested objects inside a package object lools like a practice of some kind. If it is a common practice, is there something to read about it? We might use this more - it looks like a good way to group constants and make a single place to store them.

Reply:
I might have seen this already in some codebases, generally speaking, this is just an interpretation of
> Scala provides package objects as a convenient container shared across an entire package.

(src: https://docs.scala-lang.org/tour/package-objects.html).

But mostly, having seen these literal string to be repeated over and over, I just felt to need to _do something_. 😆